pkg/uroot/builder/gbb.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
```

Comment:
2021

Reply:
it is from another file, so .. 2015-2021 or 2021/